Dockerfile

```diff
@@ -1,63 +1,73 @@
-# Configure apt, install updates and common packages, and clean up apt's cache
-ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update \
- && apt-get upgrade --assume-yes \
- && apt-get autoremove --assume-yes \
- && apt-get install --assume-yes --no-install-recommends \
-   apt-utils \
-   apt-transport-https \
-   ca-certificates \
-   ca-certificates-java \
-   software-properties-common \
-   locales \
- && apt-get install --assume-yes --no-install-recommends \
-   curl \
-   psmisc \
-   git \
-   build-essential \
-   jq \
- && apt-get install --assume-yes --no-install-recommends \
-   ffmpeg \
-   mysql-server \
-   mysql-client \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/ \
- && update-ca-certificates
+#Enable EPEL repository, adding RPM Fusion and Yarn repository
+RUN dnf install --assumeyes \
+    https://download.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm \
+    yum-utils \
+    --nogpgcheck https://mirrors.rpmfusion.org/free/el/rpmfusion-free-release-8.noarch.rpm \
+    --nogpgcheck --nogpgcheck https://mirrors.rpmfusion.org/nonfree/el/rpmfusion-nonfree-release-8.noarch.rpm \
+  && yum-config-manager --enable powertools \
+  && curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo \
+  && rpm --import https://dl.yarnpkg.com/rpm/pubkey.gpg
+
+#Install nodejs and enable module nodejs:14
+RUN dnf module reset nodejs \
+  && dnf module enable -y nodejs:14 \
+  && dnf module install -y nodejs:14 
+
+# Configure dnf, install updates and common packages, and clean up dnf's cache
+RUN dnf upgrade --assumeyes \
+  && dnf autoremove --assumeyes \
+  && dnf install --assumeyes \
+    ca-certificates \
+    curl \
+    psmisc \
+    git \
+    jq \
+    ffmpeg \
+    yarn \
+  && dnf clean dbcache \
+  && rm -rf /tmp/* /var/tmp/
 
 # Ensure locale is UTF-8
-ENV LANG       en_US.UTF-8
-ENV LC_ALL     en_US.UTF-8
-ENV LC_TYPE    en_US.UTF-8
-ENV LANGUAGE   en_US.UTF-8
-RUN echo 'en_US.UTF-8 UTF-8' >> /etc/locale.gen \
- && locale-gen \
- && dpkg-reconfigure locales
+RUN dnf install -y \
+    glibc-langpack-en \
+    glibc-locale-source \
+  && localedef --force --inputfile=en_US --charmap=UTF-8 en_US.UTF-8 \
+  && echo "LANG=en_US.UTF-8" > /etc/locale.conf \
+  && dnf clean all \
+  && rm -rf /var/cache/dnf /var/cache/yum
 
 # Create mnt dir inside the container
 RUN chown docker:docker -R /mnt
 
 # Copy package.json into container
 RUN mkdir -p /app/src/ssl \
- && chown -R docker:docker /app/src
+  && chown -R docker:docker /app/src
 COPY --chown=docker:docker package.json package-lock.json /app/src/
 
 # switch to docker user
 USER docker
 WORKDIR /app/src
 
-# Install npm dependencies
+# Install npm dependencies and update browserlist
 #ENV APP_ENV production
 #ENV RACK_ENV production
-RUN npm install
+RUN npm install \
+  && npx browserslist@latest --update-db 
+
+RUN yarn add \
+    react@latest \
+    react-dom@latest \
+    next@12 \
+    --dev @types/node
```

Comment:
For node apps it's always preferable to update the package-lock.json file so that developers are working with the same packages in dev as we use in prod.  I wanted to test it first so I didn't send you on a horrible side path, and it ended up being quite a rabbit hole, so I captured it into a commit.  I'll walk you through it next time we pair.  You'll want to pull down the branch and then add a commit to remove the yarn add